docs/docker_quickstart.md

```diff
@@ -149,6 +149,20 @@ You'll then also need to modify the `docker-compose.yml` file and uncomment the
 
 You can then run `docker-compose build` to build the docker image, and run it using the commands described above.
 
+### Troubleshooting
+
+### Docker on Windows
```

Comment:
i guess you'll want to have this "below" the troublshooting header, which means it needs to be indented with one additional hash

```suggestion
#### Docker on Windows
```